docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -519,522 +519,613 @@ Function: `non_zero_count`.
 
 These {anomaly-jobs} appear by default in the Anomaly Detection interface of the 
 {siem-guide}/machine-learning.html[SIEM app] in {kib}. They help you 
-automatically detect file system and network anomalies on your hosts. The list 
-below contains the jobs organized by `agent.type` (Auditbeat, Packetbeat, and 
-Winlogbeat).
+automatically detect file system and network anomalies on your hosts.
 
 // tag::siem-jobs[]
-[float]
-[[ootb-ml-jobs-siem-audit]]
-==== SIEM - Auditbeat
-
 linux_anomalous_network_activity_ecs::
++
+windows_anomalous_network_activity_ecs::
+
+Identifies Windows/Linux processes that do not usually use the network but have
```

Comment:
```suggestion
Identifies processes that do not usually use the network but have
```
This is maybe more style, but I think this makes the sentence easier to read. The OS specifiers in the names and in the required query I think clearly identify what OS the process is running on.

Reply:
Sure, but we'll wait for @randomuserid as the descriptions are basically his.